docs/pages/templates/typescript/threejs.mdx

```diff
@@ -189,15 +191,17 @@ export const useKeyboardMovement = () => {
 ```ts
   useEffect(() => {
     const listener = (e: KeyboardEvent) => {
-      if (e.key === "ArrowUp") {
+      if (e.key === "w") {
         moveBy(1, 0, 0);
       }
-      if (e.key === "ArrowDown") {
+      if (e.key === "s") {
         moveBy(-1, 0, 0);
-      }      .
+      }
+      .
       .
       .
       .
```

Comment:
can we use the `<CollapseCode>` component for this? that's what this is for, so we don't have to manually insert these placeholder characters

Reply:
Done